js/table/tablemanager.ts

```diff
@@ -346,6 +350,55 @@ class TableManager {
         tableView.displayRebuyOrAddonTime();
     }
 
+    public onPlayerCardsDealed(tableId: number) {
+        const tableView = this.getTableById(tableId);
+        if (!tableView) {
+            return;
+        }
+        tableView.setRoundNotificationCaption(0);
+        this.updateRoundNotificationTimer(tableId);
+    }
+
+    public onFlopDealed(tableId: number) {
+        const tableView = this.getTableById(tableId);
+        if (!tableView) {
+            return;
+        }
+        tableView.setRoundNotificationCaption(1);
+        this.updateRoundNotificationTimer(tableId);
+    }
+
+    public onTurnDealed(tableId: number) {
+        const tableView = this.getTableById(tableId);
+        if (!tableView) {
+            return;
+        }
+        tableView.setRoundNotificationCaption(2);
+        this.updateRoundNotificationTimer(tableId);
+    }
+
+    public onRiverDealed(tableId: number) {
+        const tableView = this.getTableById(tableId);
+        if (!tableView) {
+            return;
+        }
+        tableView.setRoundNotificationCaption(3);
+        this.updateRoundNotificationTimer(tableId);
+    }
+    public updateRoundNotificationTimer(tableId: number) {
+        const tableView = this.getTableById(tableId);
+        if (!tableView) {
+            return;
+        }
+        tableView.roundNotificationTimer(4);
+       var timer = timeService.setInterval(() => {
```

Comment:
Why setIterval here, and not setTimeout ? setTimeout would be called only single time